portal/views/assessment_engine.py

```diff
@@ -51,10 +52,11 @@
 @assessment_engine_api.route(
     '/patient/<int:patient_id>/assessment',
     defaults={'instrument_id': None},
-)
+    methods=('OPTIONS', 'GET'))
 @assessment_engine_api.route(
-    '/patient/<int:patient_id>/assessment/<string:instrument_id>'
-)
+    '/patient/<int:patient_id>/assessment/<string:instrument_id>',
+    methods=('OPTIONS', 'GET'))
+@crossdomain(origin='*')
```

Comment:
We'll need to protect these a little better (not allowing any domain), but still let Swagger UI to make CORS requests.

@pbugni thoughts?

Some ideas:
* Only allow in non-prod environments (`SYSTEM_TYPE != 'production'`)
* Only allow from truenth.org domains (our Swagger UI is hosted at stg.us.truenth.org/dist)

Reply:
I think we need a whitelist.  We have one in `models.client.validate_origin` - can this be extended with the known swagger hosts?